package.json

```diff
@@ -33,6 +33,7 @@
     }
   },
   "dependencies": {
-    "jest": "^24.8.0"
+    "jest": "^24.8.0",
+    "js-yaml": "^3.13.1"
```

Comment:
I believe that this was added for use in `packages/cms-cli`. If so, it should be added to the package.json local to that package. For future reference, `lerna` has some commands for managing deps.